src/xunit.console.uwp/HostedAppxTest.cs

```diff
@@ -139,24 +139,29 @@ public int Run(bool debug, TimeSpan timeout)
             Console.WriteLine("UWP Activation HRESULT: " + hr);
             if (hr == 0)
             {
-            var p = Process.GetProcessById(pid.ToInt32());
-            Console.WriteLine("Running {0} in process {1} at {2}", p.ProcessName, pid, DateTimeOffset.Now);
-            while (timer.ElapsedMilliseconds < timeout.TotalMilliseconds && !p.HasExited)
-            {
-                Thread.Sleep(1000);
+                var p = Process.GetProcessById(pid.ToInt32());
+                Console.WriteLine("Running {0} in process {1} at {2}", p.ProcessName, pid, DateTimeOffset.Now);
```

Comment:
Interpolated strings are cool. 
like: 
'''Console.WriteLine($"Running {p.ProcessName} in process {pid} at {DateTimeOffset.Now}" );